WalletWasabi.Fluent/ViewModels/Wallets/Actions/WalletActionViewModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Reactive;
+using System.Reactive.Linq;
+using ReactiveUI;
+using WalletWasabi.Fluent.ViewModels.NavBar;
+using WalletWasabi.Helpers;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Actions
+{
+	public abstract partial class WalletActionViewModel : NavBarItemViewModel
```

Comment:
Agreed with @wieslawsoltes these ActionViewModels are redundant and will be removed as we add each section.